extensions/dependencies/guppy.html

```diff
@@ -8,3 +8,27 @@
     '/third_party/static/guppy-be6f14/src/transform.xsl',
     '/third_party/static/guppy-be6f14/src/symbols.json');
 </script>
+
+<style>
+    .guppy_active {
+      box-shadow: 0 0 5px rgba(81, 203, 238, 1) !important;
```

Comment:
I agree it's overkill. Do we actually need to change the box-shadow? I'm surprised that !important would override the JS, either...